analysis/preprocess/preprocess_data.R

```diff
@@ -0,0 +1,157 @@
+# Load libraries ---------------------------------------------------------------
+
+library(magrittr)
+library(dplyr)
+library(tidyverse)
+library(lubridate)
+library(data.table)
+library(readr)
+
+# Specify command arguments ----------------------------------------------------
+
+args <- commandArgs(trailingOnly=TRUE)
+print(length(args))
+if(length(args)==0){
+  cohort_name <- "vax"
+} else {
+  cohort_name <- args[[1]]
+}
+
+# Get column names -------------------------------------------------------------
```

Comment:
Please add `print()` statements for these headings. This will allow us to see how far the script has got if any action should fail.

Reply:
Done

Reply:
Hi Zoe - I can't see the print statements, please can you confirm which commit they are in?